index.html

```diff
@@ -0,0 +1,157 @@
+    <link href="styles.css" rel="stylesheet">
+</head>
+
+<body>
+    <!-- Header Section -->
+    <section class="desktop-nav">
+        <div class="first-navbar">
+            <nav>
+                <ul>
+                    <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+                    <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+                    <li><a href="#" class=" fw-bold">English</a></li>
+                    <li><a href="#" class="fw-bold">LogIn</a></li>
+                    <li><a href="#" class="fw-bold">LogOut</a></li>
+
+                </ul>
+            </nav>
+        </div>
+        <div class="second-navbar">
+            <div>
+                <h1 class="display-4 fw-bold m-2">AHMADI GYM</h1>
+            </div>
+            <div>
+            <nav>
+                <ul>
+                    <li><a href="about.html" class="fw-bold
+                        ">About</a></li>
+                    <li><a href="#programs-section" class="fw-bold
+                        ">Program</a></li>
+                    <li><a href="#join-section" class="fw-bold">Join</a></li>
+                    <li><a href="#featured-section" class="fw-bold
+                        "></a></li>
+                    <li class="last-list-item" class="fw-bold
+                    "><a href="#" class="fw-bold">AHMADI GYM</a></li>
+
+                </ul>
+            </nav>
+            </div>
+        </div>
+    </section>
+    <section class="first-page">
+        <header class="header-sect">
+            <a href="#"><i class="fas fa-bars fa-2xl" id="hamburger-menu"></i></a>
+        </header>
+        <article class="intro-sect">
+            <p class="heading-intro fw-bold">Stop Thinking Start Doing!</p>
+            <h2 class="fw-bold">Work-Sweat-Achieve</h2>
+            <h2 class="fw-bold"> Nothing Will Work Unless You Do</h2>
+            <h2 class="fw-bold">2023</h2>
+            <p class="intro-description">The Pain You Feel Today Will Be The Strength You Feel Tomorrow.</p>
+            <h3>2023.06.28(Wed)</h3>
+            <h4>@Kabul Afghanistan</h4>
+        </article>
+    </section>
+    <!-- Programs' Section -->
+    <section class="programs-sect" id="programs-section">
+        <h2 class="main-program">AHMADI GYM Program</h2>
+        <hr class="under-line">
+        <div class="program-activities">
+            <ul>
+                <li>
+                    <i class="fas fa-atom fa-2xl comments"></i>
+                    <h4>Trainings</h4>
+                    <p>You can get everything in life you want if you will just help enough other people get what they want.</p>
+                </li>
+                <li>
+                    <i class="fas fa-burn fa-2xl comments"></i>
+                    <h4>Moments</h4>
+                    <p>You can get everything in life you want if you will just help enough other people get what they want.</p>
```

Comment:
- [ ] Currently, I notice that the icons in the main program section of the design are moving out of the card and it makes it looks different from what we have in the original design, I would kindly suggest you make the icons fit properly into the cards and be sure the design look perfect like the one we have in the original design.
- [ ] Also, the button in the `desktop  view` is quite different from the one in the Original design. The button in the design is `SEE THE WHOLE PROGRAM`.